notebooks/animate-anyone/animate-anyone.ipynb

```diff
@@ -0,0 +1,1349 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #54.            gr.Slider(label="Number of inference steps", value=30, minimum=15, maximum=100)
</pre><p>Could you add video length</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Model was converted with fixed height, width and video length to reduce memory consumption during inference.</p>